octobot_trading/personal_data/orders/order_util.py

```diff
@@ -537,3 +537,30 @@ async def get_order_size_portfolio_percent(exchange_manager, order_amount, side,
 
 def generate_order_id():
     return str(uuid.uuid4())
+
+
+async def wait_for_order_fill(order, timeout, wait_for_portfolio_update):
+    if order.is_open():
+        if order.state is None:
+            logging.get_logger(LOGGER_NAME).error(
+                f"None state on created order, impossible to wait for fill, order: {order}"
+            )
+        else:
+            try:
+                await order.state.wait_for_next_state(timeout)
+            except asyncio.TimeoutError:
+                logging.get_logger(LOGGER_NAME).error(
+                    f"Timeout while waiting for rebalance marker open order fill, order {order}"
```

Comment:
It may not be a rebalance order as it is in order_util :smile:

Reply:
:o right